x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/indicator_match_rule_suppression_serverless_essentials.cy.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewThreatIndicatorRule } from '../../../../objects/rule';
+
+import { DEFINITION_DETAILS, SUPPRESS_BY_DETAILS } from '../../../../screens/rule_details';
+
+import {
+  fillDefineIndicatorMatchRule,
+  fillAlertSuppressionFields,
+  selectIndicatorMatchType,
+  fillAboutRuleMinimumAndContinue,
+  createRuleWithoutEnabling,
+  skipScheduleRuleAction,
+  continueFromDefineStep,
+} from '../../../../tasks/create_new_rule';
+
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { getDetails } from '../../../../tasks/rule_details';
+import { CREATE_RULE_URL } from '../../../../urls/navigation';
+import { deleteAlertsAndRules } from '../../../../tasks/api_calls/common';
+
+const SUPPRESS_BY_FIELDS = ['myhash.mysha256', 'source.ip.keyword'];
+
+describe(
+  'Detection rules, Indicator Match, Alert Suppression',
+  {
+    tags: ['@serverless'],
+    env: {
+      ftrConfig: {
+        productTypes: [
+          { product_line: 'security', product_tier: 'essentials' },
+          { product_line: 'endpoint', product_tier: 'essentials' },
+        ],
+
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'alertSuppressionForIndicatorMatchRuleEnabled',
+          ])}`,
+        ],
+      },
+    },
+  },
+  () => {
+    const rule = getNewThreatIndicatorRule();
+    beforeEach(() => {
+      cy.task('esArchiverLoad', { archiveName: 'threat_indicator' });
+      cy.task('esArchiverLoad', { archiveName: 'suspicious_source_event' });
+      deleteAlertsAndRules();
+      login();
+    });
+
+    it('creates rule with per rule execution suppression for essentials license', () => {
+      login();
```

Comment:
As you are performing the login on the beforeEach hook, this one is not needed.

Reply:
@MadameSheema , thanks for review
comments have been addressed in https://github.com/elastic/kibana/pull/174241/commits/3ccd2f8fc2a2baeb900c8c545aa4f6dd921c0f79